### DISCUSSION TITLE
能否提供Armbian系统的一键安装脚本

### DISCUSSION THREAD

**Discussion by hussion** [kkkgo/PaoPaoGateWay#43] [想法]
虚拟机镜像虽然方便，但是对环境和机器还是有一定的要求。比如对于只有硬件配置不高的群辉来讲，网关虽然本身较为轻量，但是却需要运行一个虚拟机来运行网关还是有不少不必要的性能消耗，拖慢NAS本身性能，而且对留学的加解密性能上限也有一定的影响。

能否提供非虚拟机的一键安装脚本，像shellclash一样，可以比较方便的安装到Armbian盒子里？

**Comment by kkkgo** [comment]
透明代理对内核版本和模块都有依赖，只有完整的虚拟机才能保证通用。特别是比较旧型号群晖自带的内核，一般内核版本都比较低。
Armbian盒子可以刷openwrt之类，用其他插件，反正只要是FAKEIP的模式，都可以配合PaoPaoDNS使用。

**Comment by kkkgo** [comment]
现在已支持arm虚拟机镜像，参考：https://github.com/kkkgo/PaoPaoGateWay/discussions/214
